docs: new tutorial and updates related to RoundRequestor. (#54)
algorand/conduit: docs/GettingStarted.md

### Install from Source

1. Checkout the repo, or download the source, `git clone https://github.com/algorand/conduit.git && cd conduit`
2. Run `make conduit`.
3. The binary is created at `cmd/conduit/conduit`.

## Getting Started

Conduit requires a configuration file to set up and run a data pipeline. To generate an initial skeleton for a conduit
config file, you can run `./conduit init -d data`. This will set up a sample data directory with a config located at
`data/conduit.yml`.

You will need to manually edit the data in the config file, filling in a valid configuration for conduit to run.
You can find a valid config file in [Configuration.md](Configuration.md) or via the `conduit init` command.

Once you have a valid config file in a directory, `config_directory`, launch conduit with `./conduit -d config_directory`.

# Configuration and Plugins
Conduit comes with an initial set of plugins available for use in pipelines. For more information on the possible
plugins and how to include these plugins in your pipeline's configuration file see [Configuration.md](Configuration.md).

# Tutorials

> Detailed tutorials for specific topics are available.

Reviewer: suggest deleting this line.

> 

Reviewer: I think somewhere it would be useful to give a little more info about why you'd want to follow each of these tutorials. It could either be in each of these sections below, or in the intro to each tutorial itself (or both). Like for setting up Conduit to serve the Indexer API, why would I want to do that?

> ## Migrate from Indexer to Conduit

Reviewer: ```suggestion
## Migrate from the Legacy Indexer Architecture to a Conduit-backed Indexer
```

> [Information about converting from Indexer writer configurations to Conduit.](./tutorials/IndexerMigration.md)

Reviewer: ```suggestion
[How to migrate from a legacy Indexer architecture to a Conduit-backed Indexer deployment. .](./tutorials/IndexerMigration.md)
```

> ## Setup Conduit for the Indexer API

Reviewer: ```suggestion
## Set up Conduit for the Indexer API
```
I think: "setup" for the noun, "set up" for the verb.

[How to configure algod, PostgreSQL and Conduit as an Indexer API backend.](./tutorials/IndexerWriter.md)

> ## Transaction Filtering Plugin

Reviewer: ```suggestion
## Using the Transaction Filter Plugin
```
